docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -2062,6 +2093,102 @@ Override the default architecture variant of the container image.
+
+To avoid password leaking, Quadlet will set the permissions of the generated service file as 0600.
+It is recommended to do the same for `.login` unit files.
+
+Valid options for `[Login]` are listed below:
+
+| **[Login] options**                             | **podman login**                                                     |
+|-------------------------------------------------|----------------------------------------------------------------------|
+| AuthFile=/etc/registry/auth\.json               | --authfile=/etc/registry/auth\.json                                  |
+| CertDir=/etc/certs                              | --cert-dir=/etc/certs                                                |
+| LogoutOnStop=true                               | Add ExecStopPost to log out of the registry when the unit is stopped |
+| Password=mypassword                             | --password-stdin and set StandardInputText                           |
+| PodmanArgs=--compat-auth-file=/etc/compat\.json | --compat-auth-file=/etc/registry/compat\.json                        |
+| Registry=quai.io                                | podman login quai.io                                                 |
+| Secret=mysecret                                 | --secret=mysecret                                                    |
+| ServiceName=name                                | Name the systemd unit `name.service`                                 |
+| TLSVerify=false                                 | --tls-verify=false                                                   |
+| Username=myuser                                 | --username=myuser                                                    |
+
+### `AuthFile=`
+
+Path of the authentication file.
+
+This field is mandatory when linking between an `AuthFile` key of a different unit and the `.login` unit.
+
+This is equivalent to the `--authfile` option.
+
+### `CertDir=`
+
+Use certificates at path (*.crt, *.cert, *.key) to connect to the registry.
+
+This is equivalent to the Podman `--cert-dir` option.
+
+
+### `LogoutOnStop=` (defaults to `false`)
+
+When set to `true` the logout will be called when the service is stopped
+
+### `Password=`
+
+Use the password to connect to the registry.
+In order to prevent password leak, instead of using the `--password` option,
+Quadlet will set `--password-stdin` and use systemd `StandardInputText` to pass the data.
+
+
+### `PodmanArgs=`
+
+This key contains a list of arguments passed directly to the end of the `podman login` command
+in the generated file. It can be used to access Podman features otherwise unsupported by the generator. Since the generator is unaware
+of what unexpected interactions can be caused by these arguments, is not recommended to use
+this option.
+
+The format of this is a space separated list of arguments, which can optionally be individually
+escaped to allow inclusion of whitespace and other control characters.
```

Comment:
It would be nice to add some examples at the bottom of this page.  Not a necessity.